src/grid/grid.directive.ts

```diff
@@ -0,0 +1,57 @@
+import { Directive, HostBinding, Input, OnInit } from "@angular/core";
+
+@Directive({
+	selector: "[ibmGrid]"
+})
+export class GridDirective {
+	@HostBinding("class.bx--grid") baseClass = true;
+}
+
+@Directive({
+	selector: "[ibmRow]"
+})
+export class RowDirective {
+	@HostBinding("class.bx--row") baseClass = true;
+}
+
+@Directive({
+	selector: "[ibmCol]"
+})
+export class ColumnDirective implements OnInit {
+	@Input()
+	class = "";
+
+	@Input()
+	columnNumbers = {};
```

Comment:
Generally we have the decorator on the same line ... I've done up github suggestions for these, if that helps 👍 
```suggestion
	@Input() columnNumbers = {};
```